opts/file.go

```diff
@@ -53,17 +53,23 @@ func parseKeyValueFile(filename string, emptyFn func(string) string) ([]string,
 			if strings.ContainsAny(variable, whiteSpaces) {
 				return []string{}, ErrBadKey{fmt.Sprintf("variable '%s' has white spaces", variable)}
 			}
+			if len(variable) == 0 {
+				return []string{}, ErrBadKey{fmt.Sprintf("no variable name on line '%s'", line)}
+			}
 
 			if len(data) > 1 {
 				// pass the value through, no trimming
 				lines = append(lines, fmt.Sprintf("%s=%s", variable, data[1]))
 			} else {
 				var value string
+				var boolean bool
 				if emptyFn != nil {
-					value = emptyFn(line)
+					value, boolean = emptyFn(line)
```

Comment:
`boolean` is a bit odd for the variable name; perhaps change to something like `exists` or `isSet`

Reply:
Sure for the rename.

For the tests I need some assistance but I'm eager to add some / extend existing ones, I just need some guidance into.